vcs.go

```diff
@@ -110,59 +115,12 @@ func (g Git) OpenFile(revision, path string) (io.ReadCloser, error) {
 	return ioutil.NopCloser(bytes.NewReader(contents)), nil
 }
 
-func (g Git) DefaultRevision() (string, string) {
+// DefaultRevision returns the default revisions if none specified
+func (g *Git) DefaultRevision() (string, string) {
 	// Check if there's unstaged changes, if so, return dot
 	contents, _ := exec.Command("git", "--git-dir", g.dir, "ls-files", "-m").Output()
 	if len(contents) > 0 {
 		return "HEAD", "."
 	}
 	return "HEAD~1", "HEAD"
 }
-
-type fileInfo struct {
-	name string // base name of file
-	dir  bool
-}
-
-func (fi fileInfo) Name() string       { return fi.name }
-func (fi fileInfo) Size() int64        { panic("not implemented") }
-func (fi fileInfo) Mode() os.FileMode  { panic("not implemented") }
-func (fi fileInfo) ModTime() time.Time { panic("not implemented") }
-func (fi fileInfo) IsDir() bool        { return fi.dir }
-func (fi fileInfo) Sys() interface{}   { panic("not implemented") }
-
-var _ VCS = (*StrVCS)(nil)
```

Comment:
I use StrVCS in abicheck-web (https://github.com/bradleyfalzon/abicheck-web/blob/master/main.go#L46) as I didn't want to combine the two projects.